pkg/prometheus/promcfg.go

```diff
@@ -1798,7 +1807,7 @@ func (cg *ConfigGenerator) generateK8SSDConfig(
 
 		// TODO: If we want to support secret refs for k8s service discovery tls
 		// config as well, make sure to path the right namespace here.
```

Comment:
Hmm not sure what the TODO comment is about. I guess that we can remove it.

Reply:
Got it. Remove it done.